tasks/internal/push-artifacts-to-cdn-task/push-artifacts-to-cdn-task.yaml

```diff
@@ -208,6 +209,7 @@ spec:
         RUNNING_JOBS="\j" # Bash parameter for number of jobs currently running
         NUM_COMPONENTS=$(jq '.components | length' <<< "$SNAPSHOT_JSON")
 
+        set +x
```

Comment:
What's this for? We remove the `+` lines before saving the err log, so this shouldn't be needed, no?

Reply:
Probably a leftover. I will remove it.

Reply:
removed.